include/seqan/graph_align/graph_impl_align.h

```diff
@@ -666,6 +666,19 @@ getAdjacencyMatrix(Graph<Alignment<TStringSet, TCargo, TSpec> > const& g,
 
 //////////////////////////////////////////////////////////////////////////////
 
+template<typename TStringSet, typename TCargo, typename TSpec, typename TVertex, typename TVector>
+inline void
+getVertexAdjacencyVector(Graph<Alignment<TStringSet, TCargo, TSpec> > const & g,
+                         TVertex const & vertex,
+						 TVector & vectIn,
```

Comment:
Are vectIn and vectOut both modified?

Altogether this is the wrong order of arguments. In SeqAn we have out-parameters (those without const) before in-parameters (those with const). Please adapt this.